linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml

```diff
@@ -307,7 +307,7 @@
 
 - name: "mac_and_ip | wait up to 5 mins for dhcp ip address"
   shell: |
-    arp -an | grep -F {{ extract_mac_address_result.results[mac].stdout }} | cut -f 2 -d "(" | cut -f 1 -d ")"
+    /usr/sbin/arp -an | grep -F {{ extract_mac_address_result.results[mac].stdout }} | cut -f 2 -d "(" | cut -f 1 -d ")"
```

Comment:
I replied via email but I'm not entirely sure that worked so lets duplicate that reply here :)

It is my understanding that the remote user is set via {{
res_def['remote_user'] }}. In my case this is not 'root' but a user in
the libvirt and qemu groups (thus allowed to provision virtual
resources). This user apparently does not have 'arp' in $PATH.

The relevant configuration entries from my topology file are:

  uri: qemu+ssh://admin@remote-kvm-host.example.com/system
  remote_user: admin

I would ideally like to run this playbook without (too many) privileges.
However, since it is already a requirement for the remote_user to be a
member of certain groups perhaps it's not a stretch to require that
remote_user to have /usr/sbin in the $PATH.